main/EDTypesMod.F90

```diff
@@ -607,17 +608,25 @@ module EDTypesMod
         
      ! TERMINATION, RECRUITMENT, DEMOTION, and DISTURBANCE
 
```

Comment:
@ckoven , can you sign-off that changing terminated_nindivs and termination_carbonflux from 2 layers to nclmax is ok with you?  You will see changes in the code that also reflect this change in usage, and are not separated into upper and not-upper.  I changed this because it was confusing to me on first if these arrays were allocated by canopy layer or not, and for clarity and consistency sake, changed it to be by canopy layer since that is our typical convention.  We can revert it if you feel these arrays are better allocated as the binary form, but we need to give them an integer parameter that describes this dimension if so.

Reply:
From email exchange with @ckoven: 
> "great. so long as we do that everywhere, it works for me. "